lang/lua/README.md

```diff
@@ -0,0 +1,29 @@
+# Lua as Dynamic (PIE) Executable
+
+This directory stores the binary executable (ELF - *Executable and Linking Format*), dynamic library dependencies and support files to run Lua.
+The binary is PIE (*Position-Independent Executable*).
+All files are stored in their expected location in the Linux filesytem hierarchy.
+The aim is to run the executable in any Linux ELF-compatible environment: native Linux, chroot/jail, Docker, Unikraft.
+
+## Run Lua Natively on Linux
+
+The binary is located in the `usr/bin` directory.
+Run it with the command:
+
+```console
+./usr/bin/lua
+```
+
```

Comment:
The above command will use the loader and support libraries on your native system, **not** those in this directory.
Because of that, there may be version incompatibilities that give errors such as those below:

--- (replace with backticks)
./usr/bin/lua: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by ./usr/bin/lua)
./usr/bin/lua: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by ./usr/bin/lua)
---

In that case, run the command below that uses the loader and dynamic libraries from the current directory:

--- (replace with triple backticks console)
./lib64/ld-linux-x86-64.so.2 --library-path ./lib ./usr/bin/lua helloworld.lua
---